rand.go

```diff
@@ -46,3 +60,23 @@ func GenerateRandomStringURLSafe(n int) (string, error) {
 	b, err := GenerateRandomBytes(n)
 	return base64.URLEncoding.EncodeToString(b), err
 }
+
+// GenerateRandomStringV2 Generate random string adapted from
+// https://stackoverflow.com/questions/22892120/how-to-generate-a-random-string-of-a-fixed-length-in-go
+func GenerateRandomStringV2(n int) string {
```

Comment:
`GenerateRandomAlphanumeric` is more suitable IMHO, since it only include `a-z A-Z 0-9`